CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 16.3.1
+
+#### Minor Change
+
+- Allows for user to put a coma at the end of a variable reference, without consequence to the API output.
```

Comment:
Formulated like this, this sounds like a feature, while it's just robustness to user mistakes. What about something like:

> Handle the case where the user accidentally puts a coma at the end of a variable reference...